src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.TypeOfFruit;
+import java.util.List;
+import java.util.Map;
+
+public class FruitStorage {
+    private Map<TypeOfFruit, Integer> balanceMap;

```

Comment:
String would be enough for us
```suggestion
    private Map<String, Integer> balanceMap;
```